### ISSUE TITLE
I am getting HUGE log files!

### ISSUE THREAD

**Issue by RheaRainbow** [Cazzar/VTubeStudio-StreamDeck#36]
Only since very recently, I am getting HUGE 5-10GB log files in C:\Users\[my name]\AppData\Roaming\Elgato\StreamDeck\logs

Can I safely delete these files?  Is there any reason I can be getting these?

I use the small 5x3 Elgato Streamdeck with your app VTS is up to date. 
[comp.txt](https://github.com/Cazzar/VTubeStudio-StreamDeck/files/15148318/comp.txt)

**Comment by Cazzar** [comment]
Hey @RheaRainbow,

Yeah it's safe to delete those log files.
There was an issue where I previously didn't configure the logging stuff, so if you update to the latest version from either the [releases](https://github.com/Cazzar/VTubeStudio-StreamDeck/releases) page, or from the [Elgato Store](https://marketplace.elgato.com/product/vtube-studio-0ba9f87f-d5ec-44ca-98ac-6b95e6e477d3) this issue should be resolved.
It has now been made so the logs should not get generated like that any more.

Since you mentioned it is up to date, it should show "VTubeStudio 3.0.0" on the action like this: 
![image](https://github.com/Cazzar/VTubeStudio-StreamDeck/assets/994214/620313c1-0c3c-4edd-b3b3-6a68e7add23e)

**Comment by RheaRainbow** [comment]
OMG thank you so much!!

**Comment by Cazzar** [comment]
If you want to be extra sure, compare the contents of `nlog.config` to [this version](https://github.com/Cazzar/VTubeStudio-StreamDeck/blob/ddc9e5636d6651d4e339a9f4b5b9818d204d6323/streamdeck-vtubestudio/nlog.config) (you can open it in any text editor) and if they are the same, then no new log files will generate

**Comment by Cazzar** [comment]
I'm just going to close this for now as it seems it was resolved, if you want just comment and I will re-open this.

**Comment by Cazzar** [comment]
Hey @RheaRainbow 

I've just done some extra digging after a second report of it - it seems that a recent change to the StreamDeck app has caused this issue, I am going to be contacting Elgato to fix this up as the log files are a little insane there.

https://github.com/Cazzar/VTubeStudio-StreamDeck/releases/tag/3.1.0 should definately fix it up - getting this across to Elgato too.

I'm going to reopen this wile I get the update released.

**Comment by RheaRainbow** [comment]
Yeah, I found another 9GB file in the log files yesterday!! smh

It wouldn't be so bothersome if I had much more space on my C drive, but I am at the cusp of it. it sits at about 30GB left on it. I have tried to stream down as best as I can. So I have to be extra vigilant! LOL

Thank you for your hard work and dedication. Despite this bug, your plugin is amazeballs! It really helps the 2D VTuber Community! 👍✨💕
